Cargo.toml

```diff
@@ -83,6 +83,9 @@ caps = "0.5.3"
 [target.'cfg(target_os = "freebsd")'.dev-dependencies]
 sysctl = "0.4"
 
+[patch.crates-io]
+libc = { git = "https://github.com/Arnavion/libc", branch = "ktls" }
```

Comment:
[The libc PR](https://github.com/rust-lang/libc/pull/3422) has been merged, so I'm now waiting for it to be released.